docs/Release_Notes.md

```diff
@@ -49,141 +49,80 @@ cd ../ecf
 VERSION FILE CHANGES
 --------------------
 
-* `versions/run.ver` - change `version=v16.3.19` and `gfs_ver=v16.3.19`
+* `versions/run.ver` - change `version=v16.3.20`, `gfs_ver=v16.3.20`, and `obsproc_ver=v1.3`
 
 SORC CHANGES
 ------------
 
-The WAFS is no longer a submodule that is checked out within the GFS package.
-The `sorc/checkout.sh` and `Externals.cfg` checkout script no longer clone WAFS.
-The `sorc/build_all.sh` script no longer builds the WAFS code.
-The `sorc/build_gfs_wafs.sh` build script is deleted.
-The `sorc/link_fv3gfs.sh` script no longer links/copies WAFS files/execs.
+* New UFS_UTILS tag for AFWA filename change
```

Comment:
If you want some more detail, you can add how the emcsfc_snow2mdl program and its associated scripts were updated to process global AFWA snow data.

Reply:
Good suggestions...added details to release notes @ c74a4a87. Thanks!